app/javascript/__tests__/modules/listings/TableSubHeader.test.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import renderer from "react-test-renderer"
+import TableSubHeader from "../../../modules/listings/TableSubHeader"
+
+describe("TableSubHeader", () => {
+  it("renders TableSubHeader component", () => {
+    window.matchMedia = jest.fn().mockImplementation((query) => {
+      return {
+        matches: true,
+        media: query,
+        onchange: null,
+        addListener: jest.fn(),
+        removeListener: jest.fn(),
+        addEventListener: jest.fn(),
+        removeEventListener: jest.fn(),
+        dispatchEvent: jest.fn(),
+      }
+    })
```

Comment:
I'm curious, why did you add this?

Reply:
Good catch, it was a holdover from a different test file. We should be able to delete it!